aten/src/ATen/native/native_functions.yaml

```diff
@@ -3583,21 +3583,21 @@
   matches_jit_signature: True
   variants: method, function
 
-- func: pstrf(Tensor self, bool upper=True, Scalar tol=-1, *, Tensor(a!) u, Tensor(b!) piv) ->(Tensor(a!), Tensor(b!))
+- func: pstrf(Tensor self, bool upper=True, Scalar tol=-1, *, Tensor(a!) u, Tensor(b!) piv) ->(Tensor(a!) u, Tensor(b!) piv)
```

Comment:
@vishwakftw I remember you were interested in cholesky decompositions, what do you think about these names?

Personally I have no clue what these names are but I also don't know anything on the subject

Reply:
In this case, `u` seems right, because `pstrf` is meant to return upper triangular matrices.